app/javascript/app/components/sectors-agriculture/drivers-of-emissions/card-pie-chart/card-pie-chart-component.jsx

```diff
@@ -0,0 +1,131 @@
+import React, { PureComponent } from 'react';
+import { Card, PieChart, Tag, Loading, NoContent } from 'cw-components';
+import PropTypes from 'prop-types';
+import { TabletLandscape, TabletPortraitOnly } from 'components/responsive';
+import * as styles from './card-pie-chart-styles.scss';
```

Comment:
use `import styles from '...'` for scss files. `import * as styles` means that you are importing all the named exports whereas `import styles` mean you're importing the exported `default` which is here the case. The above syntax might be confusing

Reply:
Thanks for that, fixed in a few different places